src/test/v1.2/advanced/fdc3.broadcast.ts

```diff
@@ -963,11 +993,11 @@ export default () =>
         //Retrieve fdc3.instrument context
         const context = await testChannel.getCurrentContext("fdc3.instrument");
         expect(context.type).to.be.equals("fdc3.instrument", errorMessage);
-        expect(context.name).to.be.equals("History-item-2", errorMessage);
+        expect(context.name).to.be.equals("History-item-1", errorMessage);
 
         const context2 = await testChannel.getCurrentContext("fdc3.contact");
         expect(context2.type).to.be.equals("fdc3.contact", errorMessage);
-        expect(context2.name).to.be.equals("History-item-2", errorMessage);
+        expect(context2.name).to.be.equals("History-item-1", errorMessage);
```

Comment:
This needs reverting - this test is supposed to be confirming that you receive the second of two items broadcast onto the channel (which is controlled by `historyItems: 2` above).